src/context-selection/org-unit-selector-bar-item/load-offline-level.js

```diff
@@ -0,0 +1,57 @@
+// Copied from https://github.com/dhis2/ui/blob/master/components/organisation-unit-tree/src/organisation-unit-node/use-org-data/use-org-data.js
+export const QUERY_ORG_UNIT_FROM_UI = {
+    orgUnit: {
+        resource: `organisationUnits`,
+        id: ({ id }) => id,
+        params: ({ isUserDataViewFallback }) => ({
+            isUserDataViewFallback,
+            fields: ['path', 'children::size'],
+        }),
+    },
+}
+
+export const QUERY_ORG_CHILDREN_FROM_UI = {
+    orgUnit: {
+        resource: `organisationUnits`,
+        id: ({ id }) => id,
+        params: {
+            fields: ['children[id,path,displayName]'],
+        },
+    },
+}
+
+export default async function loadOfflineLevel({
+    dataEngine,
+    id,
+    offlineLevels,
+}) {
+    const variables = { id }
+    const orgData = await dataEngine.query(QUERY_ORG_UNIT_FROM_UI, {
+        variables,
+    })
```

Comment:
Why the data engine usage here?

Reply:
To make sure the request is identical to the one that the UI library does.
The service worker is supposed to cache this request and I thought that the easiest way to keep this app and the component in sync would be do duplicate what the org unit tree does in the library.